docs/Tutorial.md

```diff
@@ -545,6 +545,8 @@ This works exactly the way you expect. The lesson here is that react-admin takes
 
 ![Responsive List](./img/responsive-list.gif)
 
+**Note:** We switched to a custom API for those screenshots in order to demonstrate how to use some of the `SimpleList` component props.
```

Comment:
Why this comment? is it because the screenshot doesn't reflect the text? If so, is it worth mentioning?

Reply:
The screenshot should reflect the text but that's not the reason. The `jsonplaceholder` does not provide the fields we want to display on mobile to demonstrate the SimpleList props. As the user may be trying to reproduce our example, it will be frustrating as he/she will see undefined date and views. I thought it was nice to at least explain why he/she can't achieve the same exact result.